### DISCUSSION TITLE
How to deal with leading zeros in literals

### DISCUSSION THREAD

**Discussion by Rockaholics** [giswqs/geog-312#51] [Q&A] [answered]
![zero fill](https://github.com/user-attachments/assets/1a8ddc6a-12ad-4738-b95f-c5945c2ab3fd)
Hello, everyone.
How can I work around this problem?

**Comment by giswqs** [comment, marked as answer]
Use a dot, not comma

**Reply by Rockaholics** [reply]
Thank You
